docs/source/loggers.rst

```diff
@@ -41,6 +41,207 @@ Note:
 
 ----------
 
+Logging from a LightningModule
+------------------------------
+There are two ways to get a LightningModule to log.
+
+- The first is manual logging where you control the aggregation of metrics.
+- The second way is where you defer the aggregation to the Trainer.
+
+Training loop logging
+^^^^^^^^^^^^^^^^^^^^^
+
+Train Manual logging
+*********************
+Use this approach when you want fine-control over your logging.
+
+For the training loop, return the 'log' or 'progress_bar' keywords.
+
+- `log` sends to the `self.logger` object (tensorboard, etc)
+- `progress_bar` sends to the progress bar display
+
+.. code-block:: python
```

Comment:
I would rather set is as test-block to be sure we do not have typos and also draw that it is a class method which would be easier for new user to follow...